odh-manifests/prometheus/operator/base/prometheus.yaml

```diff
@@ -36,3 +36,35 @@ spec:
         resources:
           requests:
             storage: 100Gi
+  # Openshift OAuth Proxy config
+  listenLocal: true
+  containers:
+    - args:
+      - -provider=openshift
+      - -pass-basic-auth=false
+      - -https-address=:9091
+      - -http-address=
+      - -email-domain=*
+      - -upstream=http://localhost:9090
+      - '-openshift-sar={"resource": "namespaces", "verb": "get"}'
+      - -openshift-service-account=prometheus-k8s
+      - -openshift-ca=/etc/pki/tls/cert.pem
+      - -openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+      - -tls-cert=/etc/tls/private/tls.crt
+      - -tls-key=/etc/tls/private/tls.key
+      - -cookie-secret=SECRET
+      - -skip-auth-regex=^/metrics
+      - '-openshift-sar={"resource": "namespaces", "verb": "get", "resourceName":
```

Comment:
So only people that have get access to `opf-monitoring` can see prometheus? Is this what we want?

Reply:
yeah since it's the same with grafana and you are accessing the same data with both the tools